gateway/webhook/webhook.go

```diff
@@ -289,33 +321,11 @@ func (bt *batchWebhookTransformerT) batchTransformLoop() {
 		var payloadArr [][]byte
 		var webRequests []*webhookT
 		for _, req := range breq.batchRequest {
-			body, err := io.ReadAll(req.request.Body)
-			_ = req.request.Body.Close()
-
+			body, err := prepareRequestBody(req.request, slices.Contains(bt.webhook.config.sourceListForParsingParams, breq.sourceType), breq.sourceType, bt.webhook.config.sourceListForParsingParams)
 			if err != nil {
 				req.done <- transformerResponse{Err: response.GetStatus(response.RequestBodyReadFailed)}
 				continue
 			}
```

Comment:
Based on the following comment, we can probably just check for error here.
```suggestion
			body, err := prepareRequestBody(req.request, slices.Contains(bt.webhook.config.sourceListForParsingParams, breq.sourceType), breq.sourceType, bt.webhook.config.sourceListForParsingParams)
			if err != nil {
				req.done <- transformerResponse{Err: response.GetStatus(err.Error())}
				continue
			}
```